src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,28 @@
-export const Tabs = () => {};
+import { Tab } from '../Tab/Tab';
+
+export const Tabs = ({ tabs, findActive, active }) => {
```

Comment:
The prop `findActive` should be renamed to `onTabSelected` and `active` should be renamed to `activeTabId` to match the task description. This will ensure that the `Tabs` component receives the correct props as intended.